packages/ui/dev/scripts/generators/types.ts

```diff
@@ -1,9 +1,12 @@
 import { Member } from './generateMembers'
+import { Opening, UpcomingOpening } from './generateOpeningsAndUpcomingOpenings'
 import { Worker } from './generateWorkers'
 import { WorkingGroup } from './generateWorkingGroups'
 
 export interface Mocks {
   members: Member[]
   workingGroups: WorkingGroup[]
+  openings: Opening[]
+  upcomingOpenings: UpcomingOpening[]
   workers: Worker[]
```

Comment:
I'd rather rename `Member` to `MemberMock` etc., not to mix these types with other types that we use exactly in app